internal/lightcone/preservation/dayoneofmynewlife/dayoneofmynewlife.go

```diff
@@ -0,0 +1,58 @@
+package dayoneofmynewlife
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	mod     key.Modifier = "dayone"
+	modaura key.Modifier = "dayone-aura"
+)
+
+// Increases the wearer's DEF by 16%/18%/20%/22%/24%. After entering battle,
+// increases All-Type RES of all allies by 8%/9%/10%/11%/12%. Effects of the
+// same type cannot stack.
+func init() {
+	lightcone.Register(key.DayOneofMyNewLife, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        4,
+		Path:          model.Path_PRESERVATION,
+		Promotions:    promotions,
+	})
+
+	modifier.Register(modaura, modifier.Config{
+		Stacking: modifier.Replace,
+	})
+}
+
+func Create(engine engine.Engine, owner key.TargetID, lc info.LightCone) {
+	// wearer DEF%
+	amt := 0.14 + 0.02*float64(lc.Imposition)
+	engine.AddModifier(owner, info.Modifier{
+		Name:   mod,
+		Source: owner,
+		Stats:  info.PropMap{prop.DEFPercent: amt},
+		State:  amt,
+	})
```

Comment:
* `State` is unnecessary because this modifier has no listeners and thus nothing that is leveraging the `State` (should only be used when you need to pass information from `AddModifier` -> a listener associated with the modifier).
* When the owner of this LC dies, the buff gets removed from all team members. This modifier should have an `OnBeforeDying` listener which calls `RemoveModifierFromSource` on all characters